cmd/tikv-ctl/src/main.rs

```diff
@@ -235,6 +243,30 @@ fn main() {
             let key = unescape(&key);
             split_region(&pd_client, mgr, region_id, key);
         }
+        Cmd::Flashback { cmd: subcmd } => match subcmd {
+            FlashbackCmd::Cluster {
+                version,
+                regions,
+                start,
+                end,
+            } => {
+                let start_key = from_hex(&start).unwrap();
+                let end_key = from_hex(&end).unwrap();
+                let pd_client = get_pd_rpc_client(opt.pd, Arc::clone(&mgr));
+                flashback_whole_cluster(
+                    &pd_client,
+                    &cfg,
+                    Arc::clone(&mgr),
+                    regions.unwrap_or_default(),
+                    version,
+                    start_key,
+                    end_key,
+                );
+            }
+            FlashbackCmd::Process {} => {
+                // debug_executor.flashback_to_process();
```

Comment:
clean it?

Reply:
Oh! My fault! I will add this funxtion to show process.

Reply:
because we can check it at metrics, so remove it.